src/types/index.ts

```diff
@@ -18,6 +18,7 @@ import { PolarPlatform } from 'utils/system';
 export interface Network {
   id: number;
   name: string;
+  description: string;
```

Comment:
For users with existing networks, this value will be `undefined` when Polar starts and loads the data from `networks.json`. We do not want there to ever be a mismatch between the types and the actual runtime values. This can cause some pretty hard to find bugs. 

I would suggest checking if this description is `undefined` at startup. If so, it should be set to an empty string so that the type is always correct.